sdk/lib/stellar/sep10.rb

```diff
@@ -14,46 +14,57 @@ class SEP10
     #
     # @param server [Stellar::KeyPair] server's signing keypair (SIGNING_KEY in service's stellar.toml)
     # @param client [Stellar::KeyPair] account trying to authenticate with the server
-    # @param domain [String] service's domain to be used in the manage_data key
+    # @param home_domain [String] service's domain to be used in the manage_data key
```

Comment:
If you changed it just for the sake of maintaining similar naming to other implementations and the spec, let's revert it back please. This was a purposeful deviation since `home_domain` is a bit misleading and associates with primary Stellar domain of the organization (specifically, asset issuer), while here we have just a domain of a service requesting SEP-10 authentication.

Reply:
Got it! Yeah, I just wanted to stress the difference between "domain" and "auth domain" a little bit more with naming